M2/Macaulay2/packages/Macaulay2Doc/doc.m2

```diff
@@ -403,6 +403,33 @@ document {
      SeeAlso => { "//"}
      }
 
+doc ///
+  Key
+    (symbol /, Matrix, Number)
+    (symbol /, Matrix, RingElement)
+    (symbol /, Vector, Number)
+    (symbol /, Vector, RingElement)
+  Headline
+    scalar division
+  Usage
+    v / c
+  Inputs
+    v:{Matrix, Vector}
+    c:{Number, RingElement}
+  Outputs
+    :{Matrix, Vector}
+  Description
+    Text
+      This operation is equivalent to right scalar multiplication by the
+      multiplicative inverse of @CODE "c"@, i.e., @CODE "v * (1/c)"@.
+    Example
+      R = QQ[a,b,c,d]
+      matrix {{d, -b}, {-c, a}} / (a * d - b * c)
+  Caveat
+    The base ring of the output will be the fraction field of the base ring
+    of @CODE "v"@.
```

Comment:
I'm not sure I agree with this "Caveat". For example, `matrix{{1}}/2.` will return a matrix in `RR`...

Reply:
Hmm, good point.  Maybe something more general like "the base ring of the output will be a field containing the base ring of v"?

Reply:
I went ahead and made this change